src/e2e-app/stepper/stepper-e2e.html

```diff
@@ -0,0 +1,23 @@
+<button (click)="isLinear=!isLinear" id="toggle-linear">Enable linear</button>
```

Comment:
Rather than creating a new e2e test page, the new preference is to run e2e tests over an example from the `material-examples/` package. Check out the card tests as an example of doing this.